aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -530,12 +531,19 @@ at::Tensor PackedLinearWeightsOnednn::apply_dynamic_impl(
     x_min = t_min.item<float>();
   }
 #endif
-  const int precision = 8;
+
+#if defined(__aarch64__)
```

Comment:
Let's follow pattern introduced in https://github.com/pytorch/pytorch/blob/09a82f3d246e375b3e1288a227944108483f7b9c/aten/src/ATen/native/LinearAlgebra.cpp#L1515
as MacOS builds are produced without ACL, so I guess they would not benefit from an optimized s8s8 matmul
```suggestion
:#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
```